blocks/imagetext/imagetext.css

```diff
@@ -0,0 +1,159 @@
+.imagetext-wrapper {
+  display: flex;
+}
+
+.imagetext {
+  margin: 0.5rem;
+  border-radius: 0.5rem;
+}
+
+.imagetext-text{
+   text-align: center;
+   font-family: var(--ff-secondary);
+   font-size: calc((18 / 16) * 1rem);
+   font-style: normal;
+   font-weight: var(--fw-regular);
+   line-height: 28px;
+}
+
+.imagetext-main-container {
+  max-width: 1188px;
+  display: flex;
+  justify-content: space-between; 
+  border-radius: 8px;
+}
+
+.imagetext-left-container  {
+    width: 50%;
+    border-radius: 8px; 
+    object-fit: cover;
+}
+
+.imagetext-right-container {
+    width: 50%;
+    display: flex;
+    flex-direction: column;
+    margin: calc((20 / 16) * 1rem);
+}
+
+.banner-image , img { 
+  width: 100%;
```

Comment:
Don't directly apply the styles to the image tag or any other tag. It will affect everywhere in the site. Use proper component selector.

Reply:
Still this is not resolved